client/blocks/dismissible-card/actions.js

```diff
@@ -0,0 +1,5 @@
+import { savePreference, setPreference } from 'calypso/state/preferences/actions';
+import { getPreferenceKey } from './utils';
+
+export const dismissCard = ( preferenceName, temporary ) =>
+	( temporary ? setPreference : savePreference )( getPreferenceKey( preferenceName ), true );
```

Comment:
Nit: I know this is a function which was just moved but it is extremely hard to read. I think breaking this down into traditional if conditions will be more readable and easier to debug if it comes to it 👍